include/seastar/util/sampler.hh

```diff
@@ -147,12 +154,12 @@ private:
         // distribution.
         // We need to add 1 because that gives us the number of failures before
```

Comment:
Yes, I meant to remove it will do so now. In principle it's still +1 but then there's a -1 because we changed from > to >=, but it's probably just easiest cancel those out and leave the comment out.